diff --git a/node_modules/@jiaminghi/data-view/lib/components/scrollBoard/src/main.vue b/node_modules/@jiaminghi/data-view/lib/components/scrollBoard/src/main.vue
index ae745e7..5d4283b 100644
--- a/node_modules/@jiaminghi/data-view/lib/components/scrollBoard/src/main.vue
+++ b/node_modules/@jiaminghi/data-view/lib/components/scrollBoard/src/main.vue
@@ -12,6 +12,7 @@
         `"
         :align="aligns[i]"
         v-html="headerItem"
+        @click="$emit('headerClick', i)"
       />
     </div>
 
@@ -269,7 +270,11 @@ export default {
         data = data.map((row, i) => {
           row = [...row]
 
-          const indexTag = `<span class="index" style="background-color: ${headerBGC};">${i + 1}</span>`
+          const indexTag = `
+            <span class="index dv-scroll-board-header header-index-${i}" style="background-color: ${headerBGC};">
+              ${(i + 1).toString().length < 2 ? '0' + (i + 1) : i + 1}
+            </span>
+          `
 
           row.unshift(indexTag)
 
@@ -378,6 +383,8 @@ export default {
 
       this.animationIndex = animationIndex
       this.animationHandler = setTimeout(animation, waitTime - 300)
+
+      this.$emit('scroll', animationIndex)
     },
     stopAnimation () {
       const { animationHandler, updater } = this
